pages/ghg/ghg.md

```diff
@@ -0,0 +1,68 @@
+
+GHG Scope 1 applies to TCS Category G (Generators).
+
+### GHG Scope 2
+
+{% include categoryLabel.html label="CatO" %}
+
+Scope 2 applies to the electricity consumption of all on-site IT equipment and devices. This includes all on-premise employee devices such as computers, monitors; networking and infrastructure; and servers and storage. Energy used for lighting, heating and cooling is out of scope in the TCS, but would be accounted for in an organisation's wider GHG accounting.
+
+### GHG Scope 3
+
+Scope 3 makes up a large proportion of company emissions and the technology related Scope 3 emissions is no exception. Whilst it is an optional reporting category, there is increasing pressure on organisations to report Scope 3 emisions due to the scale of it's impact. Companies may wish to account for and report on Scope 3 emissions that are relevant to their business and have reliable information for. This, however can make it difficult to compare performance across companies.
+
+[The GHG Protocol Reporting Standard](https://ghgprotocol.org/sites/default/files/standards/ghg-protocol-revised.pdf) acknowledges that Scope 3 is a difficult and is largely inaccurate to account for. A full life cycle analysis is not required, and the company can focus its accounting effort on GHG generating activities at their discretion. It does recommend some general guidance when assessing Scope 3 emissions:
+
+1. **Describe the value chain.** For transparency, provide a general description of the value chain and associated GHG sources.
+2. **Determine which Scope 3 categories are relevant.** Only some upstream and downstream emissions are relevant to the company, for various reasons:
+  - they are (beleived to be) large relative to Scope 1 and 2 emissions
+  - they contribute to the company's GHG risk exposure and deemed critical by stakeholders
+  - there is opportunity to influence or reduce emissions
+3. **Identify partners along the value chain.** Identify any partners along the value chain that potentially contribute significant GHGs to identify sources, relevant data and calculate emissions.
+4. **Quantify Scope 3 emissions.** It is accepted that Scope 3 emissions may not be accurate due to data availability and reliability. Estimates are acceptable as long as the approach, and methodology is transparent, and that the data used is adequate and reliable.
+
+Inline with this recommended approach, the TCS describes the technology value chain across three categories, identifiying potential sources of technology emissions. The company should carefully consider these categories and sources of emissions that are most relevant to them. Using the [TCS Carbon Estimator tool](/estimator) can help identify which categories and sources may be relatively large and contribute high GHGs. Finally, the TCS provides guidance for quantifying these emissions, with details of appropriate approaches, methodology and sources of reliable data.
```

Comment:
```suggestion
In line with this recommended approach, the TCS describes the technology value chain across three categories, identifying potential sources of technology emissions. The company should carefully consider these categories and sources of emissions that are most relevant to them. Using the [TCS Carbon Estimator tool](/estimator) can help identify which categories and sources may be relatively large and contribute high GHGs. Finally, the TCS provides guidance for quantifying these emissions, with details of appropriate approaches, methodology and sources of reliable data.
```
I think you can delete the last clause of the first sentence ("identifying... emissions".) Further, "GHGs" should probably be replaced with "emissions" or similar to avoid confusion.